ros/src/aero_offboard_velocity/src/aero_drone_action.cpp

```diff
@@ -0,0 +1,188 @@
+{
+  // FCU Home position: See http://docs.ros.org/api/sensor_msgs/html/msg/NavSatFix.html
+  auto home_sub = nh_.subscribe<mavros_msgs::HomePosition>("mavros/home_position/home", 1,
+                                                           boost::bind(&AeroDrone::setHomeGeoPointCB, this, _1));
+
+  ROS_INFO("Waiting for Aero FC Home to be set...");
+  while (ros::ok() && !home_set_)
+  {
+    ros::spinOnce();
+    rate_.sleep();
+  }
+}
+
+// Callback that gets called periodically from MAVROS notifying Global Poistion of Aero FCU
+void AeroDrone::setHomeGeoPointCB(const mavros_msgs::HomePositionConstPtr& home)
+{
+  home_ = *home;
+  home_set_ = true;
+  ROS_INFO("Received Home (WGS84 datum): %lf, %lf, %lf", home_.geo.latitude, home_.geo.longitude, home_.geo.altitude);
+}
+
+void AeroDrone::getAltitude()
+{
+  ros::Subscriber altitude_sub =
+      nh_.subscribe<mavros_msgs::Altitude>("/mavros/altitude", 1, boost::bind(&AeroDrone::getAltitudeCB, this, _1));
+  while (ros::ok() && !altitude_received_)
+  {
+    ros::spinOnce();
+    rate_.sleep();
+  }
+}
+
+void AeroDrone::getAltitudeCB(const mavros_msgs::AltitudeConstPtr& altitude)
+{
+  altitude_ = *altitude;
+  altitude_received_ = true;
+  altitude_in_amsl_ = altitude_.amsl;
+}
+
+float AeroDrone::toRadFromDeg(float deg)
+{
+  return (float)(deg / 180.0f * M_PI);
+}
+
+void AeroDrone::setOffboardVelocityBody(float vx, float vy, float vz, float yaw_rate)
+{
+  mavros_msgs::PositionTarget pos{};
+  pos.coordinate_frame = mavros_msgs::PositionTarget::FRAME_BODY_NED;
+
+  pos.type_mask = mavros_msgs::PositionTarget::IGNORE_PX | mavros_msgs::PositionTarget::IGNORE_PY |
+                  mavros_msgs::PositionTarget::IGNORE_PZ | mavros_msgs::PositionTarget::IGNORE_AFX |
+                  mavros_msgs::PositionTarget::IGNORE_AFY | mavros_msgs::PositionTarget::IGNORE_AFZ |
+                  mavros_msgs::PositionTarget::FORCE | mavros_msgs::PositionTarget::IGNORE_YAW;
+  pos.velocity.x = vx;
+  pos.velocity.y = vy;
+  pos.velocity.z = vz;
+  pos.yaw_rate = toRadFromDeg(yaw_rate);
+  set_vel_pub_.publish(pos);
+}
+
+void AeroDrone::setOffboardVelocityBody(float vx, float vy, float vz, float yaw_rate, int count)
+{
+  for (; count > 0; count--)
+  {
+    AeroDrone::setOffboardVelocityBody(vx, vy, vz, yaw_rate);
+    ros::Duration(0.01).sleep();
+  }
+}
+
+void AeroDrone::setOffboardVelocityNed(float vx, float vy, float vz, float yaw)
+{
+  mavros_msgs::PositionTarget pos;
+  pos.coordinate_frame = mavros_msgs::PositionTarget::FRAME_LOCAL_NED;
+
+  pos.type_mask = mavros_msgs::PositionTarget::IGNORE_PX | mavros_msgs::PositionTarget::IGNORE_PY |
+                  mavros_msgs::PositionTarget::IGNORE_PZ | mavros_msgs::PositionTarget::IGNORE_AFX |
+                  mavros_msgs::PositionTarget::IGNORE_AFY | mavros_msgs::PositionTarget::IGNORE_AFZ |
+                  mavros_msgs::PositionTarget::FORCE | mavros_msgs::PositionTarget::IGNORE_YAW_RATE;
+  pos.position.x = 0.0f;
```

Comment:
Remove explicit initialization of `0.f` and compensate it by using [value initialization](http://en.cppreference.com/w/cpp/language/value_initialization) method as below:
```
mavros_msgs::PositionTarget pos{};
```